src/routes/signin/passwordless/email.ts

```diff
@@ -115,7 +115,6 @@ export const signInPasswordlessEmailHandler: RequestHandler<
       redirectTo: encodeURIComponent(redirectTo),
       locale: user.locale ?? ENV.AUTH_LOCALE_DEFAULT,
       serverUrl: ENV.AUTH_SERVER_URL,
-      clientUrl: ENV.AUTH_CLIENT_URL,
```

Comment:
I don't think we should delete this env var for emails because:

- It's a breaking change.
- It's useful to create generic links to the frontend app in the email.

Reply:
sure, I reverted
this variable could however be inconsistent with redirectTo when it points to another client url

Reply:
I think of `client_url` more as the default URL to your frontend apps. It could be used to add generic links like:

```
See our privacy policy: ${client_url}/privacy_policy
```

Reply:
then `AUTH_CLIENT_URL` could be renamed to `AUTH_DEFAULT_CLIENT_URL` at a later stage